src/open_inwoner/js/components/search/filter-mobile.js

```diff
@@ -0,0 +1,35 @@
+export class FilterMobile {
+  static selector = '.filter__mobile--toggle'
+
+  constructor(node) {
+    this.node = node
+    this.filterList = node.closest('.filter') // Find the closest parent with class 'filter'
+    this.isSmallscreen = window.innerWidth < 768 // Check if it's a mobile screen
```

Comment:
In general it is a bad idea to put magic numbers (and decisions based on them) in random files. We don't want every component all have their own definitions of what is a small screen (mobile screen?).